modules/nodes-pods-autoscaling-custom-gather.adoc

```diff
@@ -0,0 +1,185 @@
+    │   └── horizontalpodautoscalers.yaml
+    ├── batch
+    │   ├── cronjobs.yaml
+    │   └── jobs.yaml
+    ├── build.openshift.io
+    │   ├── buildconfigs.yaml
+    │   └── builds.yaml
+    ├── core
+    │   ├── configmaps.yaml
+    │   ├── endpoints.yaml
+    │   ├── events.yaml
+    │   ├── persistentvolumeclaims.yaml
+    │   ├── pods.yaml
+    │   ├── replicationcontrollers.yaml
+    │   ├── secrets.yaml
+    │   └── services.yaml
+    ├── discovery.k8s.io
+    │   └── endpointslices.yaml
+    ├── image.openshift.io
+    │   └── imagestreams.yaml
+    ├── k8s.ovn.org
+    │   ├── egressfirewalls.yaml
+    │   └── egressqoses.yaml
+    ├── keda.sh
+    │   ├── kedacontrollers
+    │   │   └── keda.yaml
+    │   ├── scaledobjects
+    │   │   └── example-scaledobject.yaml
+    │   └── triggerauthentications
+    │       └── example-triggerauthentication.yaml
+    ├── monitoring.coreos.com
+    │   └── servicemonitors.yaml
+    ├── networking.k8s.io
+    │   └── networkpolicies.yaml
+    ├── openshift-keda.yaml
+    ├── pods
+    │   ├── custom-metrics-autoscaler-operator-58bd9f458-ptgwx
+    │   │   ├── custom-metrics-autoscaler-operator
+    │   │   │   └── custom-metrics-autoscaler-operator
+    │   │   │       └── logs
+    │   │   │           ├── current.log
+    │   │   │           ├── previous.insecure.log
+    │   │   │           └── previous.log
+    │   │   └── custom-metrics-autoscaler-operator-58bd9f458-ptgwx.yaml
+    │   ├── custom-metrics-autoscaler-operator-58bd9f458-thbsh
+    │   │   └── custom-metrics-autoscaler-operator
+    │   │       └── custom-metrics-autoscaler-operator
+    │   │           └── logs
+    │   ├── keda-metrics-apiserver-65c7cc44fd-6wq4g
+    │   │   ├── keda-metrics-apiserver
+    │   │   │   └── keda-metrics-apiserver
+    │   │   │       └── logs
+    │   │   │           ├── current.log
+    │   │   │           ├── previous.insecure.log
+    │   │   │           └── previous.log
+    │   │   └── keda-metrics-apiserver-65c7cc44fd-6wq4g.yaml
+    │   └── keda-operator-776cbb6768-fb6m5
+    │       ├── keda-operator
+    │       │   └── keda-operator
+    │       │       └── logs
+    │       │           ├── current.log
+    │       │           ├── previous.insecure.log
+    │       │           └── previous.log
+    │       └── keda-operator-776cbb6768-fb6m5.yaml
+    ├── policy
+    │   └── poddisruptionbudgets.yaml
+    └── route.openshift.io
+        └── routes.yaml
+----
+====
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* The {product-title} CLI (`oc`) installed.
+
+.Procedure
+
+. Navigate to the directory where you want to store the `must-gather` data.
++
+[NOTE]
+====
+If your cluster is using a restricted network, you must take additional steps. If your mirror registry has a trusted CA, you must first add the trusted CA to the cluster. For all clusters on restricted networks, you must import the default `must-gather` image as an image stream.
```

Comment:
```suggestion
If your cluster is using a restricted network, you must take additional steps. If your mirror registry has a trusted CA, you must first add the trusted CA to the cluster. For all clusters on restricted networks, you must import the default `must-gather` image as an image stream by running the following command:
```

Reply:
Good catch. I copied that admonition from the [proper must-gather topic](https://docs.openshift.com/container-platform/4.12/support/gathering-cluster-data.html#support_gathering_data_gathering-cluster-data) and didn't think twice. I might just fix it over there, too.